pkg/certs/cert-inspection/certgraphapi/type_registry.go

```diff
@@ -0,0 +1,98 @@
+package certgraphapi
+
+import "fmt"
+
+type PKIRegistryInfo struct {
+	CertificateAuthorityBundles []PKIRegistryInClusterCABundle    `json:"certificateAuthorityBundles"`
+	CertKeyPairs                []PKIRegistryInClusterCertKeyPair `json:"certKeyPairs"`
+}
+
+type PKIRegistryInClusterCertKeyPair struct {
+	SecretLocation InClusterSecretLocation `json:"secretLocation"`
+
+	CertKeyInfo PKIRegistryCertKeyPairInfo `json:"certKeyInfo"`
+}
+
+type PKIRegistryCertKeyPairInfo struct {
+	OwningJiraComponent string `json:"owningJiraComponent"`
+	Description         string `json:"description"`
+
+	//CertificateData PKIRegistryCertKeyMetadata
+}
+
+type PKIRegistryInClusterCABundle struct {
+	ConfigMapLocation InClusterConfigMapLocation `json:"configMapLocation"`
+
+	CABundleInfo PKIRegistryCertificateAuthorityInfo `json:"certificateAuthorityBundleInfo"`
+}
+
+type PKIRegistryCertificateAuthorityInfo struct {
+	OwningJiraComponent string `json:"owningJiraComponent"`
+	Description         string `json:"description"`
+
+	//CertificateData []PKIRegistryCertKeyMetadata
+}
+
+type RegistrySecretByNamespaceName []PKIRegistryInClusterCertKeyPair
```

Comment:
move functions to a helper package to better match "normal" APIs.  We keep them separate to handle dependency chains better.